.github/workflows/ci.yml

```
@@ -0,0 +1,96 @@
+name: CI
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)